gtk4/src/lib.rs

```diff
@@ -17,6 +17,10 @@ extern crate libc;
 #[macro_use]
 extern crate bitflags;
 extern crate once_cell;
+#[doc(hidden)]
+#[allow(unused_imports)]
```

Comment:
` #[allow(unused_imports)]` shouldn't be needed, maybe that's part of the problem :)

Reply:
I re-exported the macros and was able to get it to compile, but the warning persists unless I do this; Should I plug `#[allow(unused_imports)]` back in?

Reply:
I guess so